tests/storage/test_raft_storage.rs

```diff
@@ -68,6 +67,31 @@ fn test_raft_storage() {
 }
 
 #[test]
+fn test_raft_storage_store_not_match() {
+    let (_cluster, storage, mut ctx) = new_raft_storage();
+    let key = make_key(b"key");
+    assert_eq!(storage.get(ctx.clone(), &key, 5).unwrap(), None);
+    storage.prewrite(ctx.clone(),
+                  vec![Mutation::Put((key.clone(), b"value".to_vec()))],
+                  b"key".to_vec(),
+                  10)
+        .unwrap();
+    storage.commit(ctx.clone(), vec![key.clone()], 10, 15).unwrap();
+    assert_eq!(storage.get(ctx.clone(), &key, 20).unwrap().unwrap(),
+               b"value".to_vec());
+
+    // Test store not match.
+    let mut peer = ctx.get_peer().clone();
+    let store_id = peer.get_store_id();
+    peer.set_store_id(store_id + 1);
+    ctx.set_peer(peer);
+    assert!(storage.get(ctx.clone(), &key, 20).is_err());
```

Comment:
Thanks very much. I read some similar tests in the  directory and it seems we haven't  check any error content, so I copied the style. Actually, I think it's a good idea to check the error content and try this style to improve the system, so I'd like to take your advice in current test.